--- C:/Users/DaLaw2/.rustup/toolchains/nightly-x86_64-pc-windows-msvc/lib/rustlib/src/rust/library/std/src/sys/alloc/mod.rs	2025-08-26 21:28:41.848491300 +0800
+++ D:/Code/Rust/async_gpu/patched-std/library/std/src/sys/alloc/mod.rs	2026-03-12 01:27:09.494602200 +0800
@@ -40,6 +40,7 @@
     target_arch = "sparc64",
     target_arch = "riscv64",
     target_arch = "wasm64",
+    target_arch = "nvptx64",
 )) {
     16
 } else {
@@ -101,4 +102,7 @@
     target_os = "zkvm" => {
         mod zkvm;
     }
+    target_os = "cuda" => {
+        mod cuda;
+    }
 }
