--- a/library/std/src/sys/thread_local/mod.rs
+++ b/library/std/src/sys/thread_local/mod.rs
@@ -24,6 +24,11 @@
 )]
 
 cfg_select! {
+    target_os = "cuda" => {
+        mod gpu_threads;
+        pub use gpu_threads::{EagerStorage, LazyStorage, thread_local_inner};
+        pub(crate) use gpu_threads::{LocalPointer, local_pointer};
+    }
     any(
         all(target_family = "wasm", not(target_feature = "atomics")),
         target_os = "uefi",
@@ -118,6 +123,7 @@
         any(
             target_os = "hermit",
             target_os = "xous",
+            target_os = "cuda",
         ) => {
             // `std` is the only runtime, so it just calls the destructor functions
             // itself when the time comes.
